test/test_main.py

```diff
@@ -585,9 +634,9 @@ def test_multiple_events_handled(self):
 
             self.assertEqual(list(state.observed_event_types), [event_spec.event_type.__name__])
 
-            if event_spec.event_name in expected_event_data:
-                self.assertEqual(state[f"{event_spec.event_name}_data"],
-                                 expected_event_data[event_spec.event_name])
+            if expected_event_data:
```

Comment:
Why the change from `if event_spec.event_name in expected_event_data` to just `if expected_event_data`?

Reply:
Because the test did not work at all :). See the first bullet point in the PR description.

Reply:
Ah, that's that one -- thanks.